packages/react-components/react-nav-preview/src/components/Nav/useNav.ts

```diff
@@ -22,16 +19,28 @@ import type { NavItemRegisterData, NavItemValue } from '../NavContext.types';
  * @param ref - reference to root HTMLDivElement of Nav
  */
 export const useNav_unstable = (props: NavProps, ref: React.Ref<HTMLDivElement>): NavState => {
-  const { onNavItemSelect } = props;
+  const { onNavItemSelect, onNavCategoryItemToggle } = props;
 
   const innerRef = React.useRef<HTMLElement>(null);
 
+  const [openItems, setOpenItems] = useControllableState({
+    // normalizeValues(controlledOpenItems), [controlledOpenItems])
+    state: React.useMemo(() => normalizeValues(), []),
+    defaultState: () => [], // initializeUncontrolledOpenItems({ defaultOpenItems }),
+    initialState: [],
+  });
```

Comment:
Are you planning to add props in the future to be able to control which items are open? Why is the `Nav` component controlling this instead of each `NavCategoryItem` tracking its own open state?